inc/class-importer.php

```diff
@@ -330,4 +329,27 @@ public function get_markdown_source( $post_id ) {
 
 		return $markdown_source;
 	}
+
+	/**
+	 * Convert the source URL to a download URL.
+	 *
+	 * @param string $source_url Source URL from {@see get_markdown_source}.
+	 * @param int    $post_id Post ID being fetched.
+	 * @return string URL to download source from.
+	 */
+	protected function get_markdown_download_source( $source_url, $post_id ) {
+		$source_url = preg_replace( '#https?://github\.com/([^/]+/[^/]+)/blob/(.+)#', 'https://raw.githubusercontent.com/$1/$2', $source_url );
+		$source_url = add_query_arg( 'v', time(), $source_url );
+		return $source_url;
+	}
+
+	/**
+	 * Get headers to send to Markdown source URL.
+	 *
+	 * @param int $post_id Post ID being fetched.
+	 * @return array Headers to pass to wp_remote_request()
+	 */
+	protected function get_markdown_source_headers( $post_id ) {
+		return array();
```

Comment:
Looks like this needs finishing off 👯‍♂️

Reply:
Lolz, ignore me!

Reply:
Yep, this is just here for subclass overrides. :)